src/devices/ovs/nm-ovsdb.c

```diff
@@ -690,6 +690,9 @@ _delete_interface (NMOvsdb *self, json_t *params, const char *ifname)
 		_expect_ovs_bridges (params, priv->db_uuid, bridges);
 		_set_ovs_bridges (params, priv->db_uuid, new_bridges);
 	}
+
+	json_decref (bridges);
+	json_decref (new_bridges);
```

Comment:
can we add and use a `nm_auto_decref_json` macro?

Reply:
Done.